data/migrations/22.lua

```diff
@@ -1,3 +1,5 @@
 function onUpdateDatabase()
-	return false
+	print("> Updating database to version 23 (Fix skulltime)")
+	db.query("ALTER TABLE players MODIFY skulltime BIGINT(20) NOT NULL DEFAULT '0'")
```

Comment:
Since it's a BIGINT, the `DEFAULT 0` should be unquoted (the `0` specifically).

Reply:
I didn't really know that there was a difference, Thanks for pointing it out!,